docs/administration/database.md

```diff
@@ -1,4 +1,362 @@
+        <Resource name="exo-jpa_portal" ...
+        username="sa" password="" driverClassName="org.hsqldb.jdbcDriver" .../>
+      ```
+
+   1. Add a new one. For MySQL as an example, you will just need to copy the sample in `conf/server-mysql.xml`:
+
+      ``` xml
+        <!-- eXo IDM Datasource for portal -->
+          <Resource name="exo-idm_portal" auth="Container" type="javax.sql.DataSource"
+          ...
+          username="plf" password="plf" driverClassName="com.mysql.jdbc.Driver" url="jdbc:mysql://localhost:3306/plf?autoReconnect=true" />
+        <!-- eXo JCR Datasource for portal -->
+          <Resource name="exo-jcr_portal" auth="Container" type="javax.sql.DataSource"
+          ...
+          username="plf" password="plf" driverClassName="com.mysql.jdbc.Driver" url="jdbc:mysql://localhost:3306/plf?autoReconnect=true" />
+        <!-- eXo JPA Datasource for portal -->
+          <Resource name="exo-jpa_portal" auth="Container" type="javax.sql.DataSource"
+          ...
+          username="plf" password="plf" driverClassName="com.mysql.jdbc.Driver" url="jdbc:mysql://localhost:3306/plf?autoReconnect=true&amp;characterEncoding=utf8" />
+      ```
+
+   1. Edit username, password, url (host, port and database name).Besides MySQL, if you are using Enterprise Edition, you will
+        find the samples for other RDBMSs in `conf/server-*.xml`.
+
+   1. Append this character encoding to the url in case your database character set is `utf8`. For example, in MySQL (this is different between RDBMSs):
+
+     ```TEXT
+        url="jdbc:mysql://localhost:3306/plf?autoReconnect=true&amp;characterEncoding=utf8"
+     ```
+
+2. Set the SQL Dialect if necessary. This step is not mandatory because the dialect is auto-detected in most cases. You only need to take care of it for some particular RDBMSs:
+   - For JCR, only when you are using MySQL and database character set `utf8`, you need to edit `gatein/conf/exo.properties` file to have:
+
+     ```properties
+        exo.jcr.datasource.dialect=MySQL-UTF8
+     ```
+
+   - For IDM, eXo Platform detects automatically the dialect for RDBMSs listed [here](http://docs.jboss.org/hibernate/orm/4.1/manual/en-US/html_single/#configuration-optional-dialects). Only when your RDBMS is **not** in the list, for example *Postgres Plus Advanced Server 9.2*, you will need to edit `gatein/conf/exo.properties` file to have:
+
+     ```properties
+       hibernate.dialect=org.hibernate.dialect.PostgresPlusDialect
+     ```
+
+     Or for *Oracle Database 12c*:
+
+     ```properties
+        hibernate.dialect=org.hibernate.dialect.Oracle10gDialect
+     ```
+
+3. Download the JDBC driver for Java and install it to `$PLATFORM_TOMCAT_HOME/lib`.
+
+::: tip
+Normally you can find out an appropriate driver for your JDK from your database vendor website. For example, for MySQL: <http://dev.mysql.com/downloads/connector/j/>, and for Oracle: <http://www.oracle.com/technetwork/database/features/jdbc/index-091264.html>.
```

Comment:
should we add the possibility to install the related addon ?

Reply:
yes I think this will be helpful